<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogViewForm</class>
 <widget class="QWidget" name="CatalogViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_Setting">
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_showDatasheet">
     <property name="text">
      <string>Show Datasheet</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox_Supplyer"/>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCustomTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>src/qcustomtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
